### ISSUE TITLE
ReferenceError: fontFamily is not defined

### ISSUE THREAD

**Issue by motifdev** [felixmariotto/three-mesh-ui#111]
I am getting the below font error and cant see any fonts when I use the library with plain javascript (loading as module)

ReferenceError: fontFamily is not defined

``` ​
import * as THREE from 'https://cdn.skypack.dev/three'; 
import ThreeMeshUI from 'https://cdn.skypack.dev/three-mesh-ui'
var posterContainer = new ThreeMeshUI.Block({ 
	width: 1.2,
	height: 0.5,
	padding: 0.05,
	justifyContent: 'center',
	alignContent: 'left', 
	fontTexture: 'https://unpkg.com/three-mesh-ui/examples/assets/Roboto-msdf.png',  
	fontFamily: 'https://unpkg.com/three-mesh-ui/examples/assets/Roboto-msdf.json'
​}); 
```

Tried with local fonts and importing from unpkg without success. Tried to set the font defaults and running the library from my own npm account withoput success. Hardly made it work on nextjs client-side nextjs by loading the library dynamically bacause of fonts.
Looking at the other issues, 50% of all open issues are because of fonts..so I request an update on this.

**Comment by felixmariotto** [comment]
Looks like these two links return valid files:
https://unpkg.com/three-mesh-ui/examples/assets/Roboto-msdf.png
https://unpkg.com/three-mesh-ui/examples/assets/Roboto-msdf.json

And I don't know how you can get the error `ReferenceError: fontFamily is not defined` with the code you provided, since there is no fontFamily variable in it.

> Looking at the other issues, 50% of all open issues are because of fonts..so I request an update on this.

Please elaborate, which issues do you think are related to your problem ?

**Comment by motifdev** [comment]
Hi,
Thank you for the quick reply. I tried to sort it yesterday without luck and still trying to find a way. I think the issue is because I'm doing server side rending ...If I remove fontTexture and family postercontainer gets filled and I can see the mesh but without the fonts loaded.  Fonts do work on my nextjs client side only if I load them dynamically but I have use the server-side for this case.

This is the code. Script is loaded under nextjs _document.tsx like this (tried to load from cdn and my own modified three-mesh-ui' npm pack  as well)


<script type="module"  src="http://localhost:3000/aframe/space-loader.js"></script>

```

import * as THREE from 'https://cdn.skypack.dev/three';  
import ThreeMeshUI from 'https://cdn.skypack.dev/three-mesh-ui' 

var posterContainer = new ThreeMeshUI.Block({ 
	width: 1.2,
	height: 0.5,
	padding: 0.05,
	justifyContent: 'center',
	alignContent: 'left', 
	fontTexture: 'https://unpkg.com/three-mesh-ui/examples/assets/Roboto-msdf.png',  
	fontFamily: 'https://unpkg.com/three-mesh-ui/examples/assets/Roboto-msdf.json'
});  
```

This is the error I get:
```
Unhandled Runtime Error
ReferenceError: fontFamily is not defined

Call Stack
Object.onLoad
https://cdn.skypack.dev/-/three-mesh-ui@v6.0.0-Jxy62uI7ZYBxWwanuSjj/dist=es2019,mode=imports/optimized/three-mesh-ui.js (358:35)
XMLHttpRequest.<anonymous>
https://cdn.skypack.dev/-/three@v0.132.2-dLPTyDAYt6rc6aB18fLm/dist=es2019,mode=imports/optimized/three.js (21778:24)
```

**Comment by NielsDewitte** [comment]
Hi,

I was playing around with this yesterday, and ran into a very similar issue.

Seems to happen in the JSFiddle you provide in the quick start.
![image](https://user-images.githubusercontent.com/9057405/147846659-433b3524-082e-4c1d-9220-69a95344ab55.png)

If I go down the stack it seems to happen in a function called loadFontJSON 
![image](https://user-images.githubusercontent.com/9057405/147846670-46e640f9-a29f-42d5-be5f-00534206098a.png)

![image](https://user-images.githubusercontent.com/9057405/147846766-c94ead10-7241-4d1f-a0c5-1361be12542c.png)

**Comment by felixmariotto** [comment]
OK sorry about this, it was a bug introduced by a very recent PR (#106).
It's fixed in version 6.0.1 : https://jsfiddle.net/felixmariotto/y81rf5t2/44/

**Comment by swingingtom** [comment]
thanks @felixmariotto
